src/app/domain/components/header/header.component.html

```diff
@@ -11,7 +11,7 @@
     }" ></app-hmcts-global-header>
 
     <app-hmcts-primary-navigation [label]="" [items]="[{
-        text: 'Dashboard',
+        text: 'Your cases',
```

Comment:
wouldn't it be better if you move this to a variable into the TS file?

Reply:
it would but i can't be refactoring the whole app in this PR... this has been there I just changed the title... feel free to do it

Reply:
OK.

Reply:
anyhow have done it...